src/core/include/openvino/core/model.hpp

```diff
@@ -536,6 +536,9 @@ class OPENVINO_API Model : public std::enable_shared_from_this<Model> {
     /// model and registers them, otherwise checks all the Parameters are registered.
     void prerequirements(bool detect_variables, bool detect_parameters);
 
+    // Check that a Node argument for ctor isn't nullptr.
+    static const std::shared_ptr<Node>& verify_node(const std::shared_ptr<Node>& node);
+
```

Comment:
I don't think that we need to extend public API
```suggestion
```

Reply:
done